ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -1,71 +1,31 @@
 SectionVerb {
 	EnableSequence [
-		disdevall ""
+		#disdevall ""
```

Comment:
The Left / Right DAC switches are already set in the BootSequence (sof-essx8336.conf) file. Also, keep disdevall line uncommented. It will execute all DisableSequence commands for all defined UCM devices.